src/components/ApplyDeveloper.tsx

```diff
@@ -142,11 +155,11 @@ const Developer: React.FC = () => {
           >
             <Step key={1} active>
               <StepLabel>
-                Prepare your resume, read through our website!
+                Prepare your resume and read through our website
```

Comment:
Do we want punctuation here? Havent looked at the site yet so I'll come back to this

Reply:
I think we might